service/src/main/java/com/theokanning/openai/service/OpenAiService.java

```diff
@@ -4,10 +4,7 @@
 import com.fasterxml.jackson.databind.DeserializationFeature;
 import com.fasterxml.jackson.databind.ObjectMapper;
 import com.fasterxml.jackson.databind.PropertyNamingStrategy;
-import com.theokanning.openai.DeleteResult;
-import com.theokanning.openai.OpenAiApi;
-import com.theokanning.openai.OpenAiError;
-import com.theokanning.openai.OpenAiHttpException;
+import com.theokanning.openai.*;
```

Comment:
Unnecessary changes to imports of `OpenAiService`.  Unrelated to the purposed feature. Changes should be reverted.